packages/parcel-transformer/src/__tests__/transformer.parceltest.tsx

```diff
@@ -0,0 +1,84 @@
+import { join } from 'path';
+
+import Parcel, { createWorkerFarm } from '@parcel/core';
+import { MemoryFS } from '@parcel/fs';
+
+const rootPath = join(__dirname, '..', '..', '..', '..');
+const fixtureRoot = join(rootPath, 'fixtures/parcel-transformer-test-app');
+const extractionFixtureRoot = join(rootPath, 'fixtures/parcel-transformer-test-extract-app');
+
+const workerFarm = createWorkerFarm();
+
+afterAll(() => {
+  workerFarm.end();
+});
+
+describe('transformer without extraction', () => {
+  const outputFS = new MemoryFS(workerFarm);
+  const parcel = new Parcel({
+    config: join(fixtureRoot, '.parcelrc'),
+    entries: [join(fixtureRoot, 'src', 'index.html')],
+    workerFarm,
+    outputFS,
+    targets: {
+      default: {
+        distDir: join(fixtureRoot, 'dist'),
+      },
+    },
+  });
+
+  it('transforms assets with babel plugin', async () => {
+    const { changedAssets } = await parcel.run();
+
+    const asset = Array.from(changedAssets.values()).find(
+      (asset) => asset.filePath === join(fixtureRoot, 'src/index.js')
+    );
+    expect(asset).toBeDefined();
+
+    const code = await asset?.getCode();
+    expect(code).toInclude('/* index.js generated by @compiled/babel-plugin v0.0.0 */');
```

Comment:
Are you able to take an inline snapshot? If there's too much code, maybe trim it to where you need